src/main/java/com/questhelper/requirements/ItemRequirement.java

```diff
@@ -118,6 +123,15 @@ public void addAlternates(Integer... alternates)
 		Collections.addAll(this.alternates, alternates);
 	}
 
+	public void addAlternateItem(int itemID, String text)
+	{
+		this.itemNameMap.put(itemID, text);
+		if (!alternates.contains(itemID))
+		{
+			alternates.add(itemID);
+		}
+	}
+
```

Comment:
Basically, if you directly add alternates you will use the default name provided via the constructor. If you add alternates via this method it will update the overlay accordingly.
The quantity of required item(s) stays the same.